app/models/reservation.rb

```diff
@@ -118,9 +120,15 @@ def self.relay_in_progress
     where("actual_start_at IS NOT NULL AND actual_end_at IS NULL")
   end
 
+  scope :not_offline, -> { where(type: nil) }
```

Comment:
OK, makes sense. I didn't realize that's how it worked. One idea I found on stackoverflow is to set the default value in the DB to "Reservation", then it would always be there. We don't need to do it, just an idea.